src/__tests__/fixtures/exclamation-value.expanded.css

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
The current increase in a test case need to add four files, which is too painful.